fundamentals/installation.md

```diff
@@ -2,106 +2,125 @@
 
 Welcome to the **napari** installation tutorial!
 
-This tutorial will teach you how to do a clean install of **napari**. It is aimed at people that just want to use napari. For people also interested in contributing to napari please check our [contributing guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md) for more advanced installation procedures. At the end of the tutorial you should have napari successfully installed on your computer and be able to make the napari viewer appear.
+This tutorial will teach you how to do a clean install of **napari**. It is
+aimed at people that just want to use napari. For people also interested in
+contributing to napari please check our [contributing
+guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
+for more advanced installation procedures. At the end of the tutorial you should
+have napari successfully installed on your computer and be able to make the
+napari viewer appear.
 
 ## installation
 
-**napari** can be installed on most macOS, Linux, and Windows systems with Python 3.6 or 3.7. There are three different ways to install napari.
+### from pip, with "batteries included"
 
-### install the latest release from pip
- The simplest option is to install the latest release on PyPi by calling
+napari can be installed on most macOS, Linux, and Windows systems with
+Python 3.6, 3.7 and 3.8 using pip:
 
 ```sh
-$ pip install napari
+pip install napari[all]
+```
+
+Note: while not strictly required, it is *highly* recommended to install
+napari into a clean virtual environment using an environment manager like
+[conda](https://docs.conda.io/en/latest/miniconda.html) or
+[venv](https://docs.python.org/3/library/venv.html).  For example, with `conda`:
+
+```sh
+conda create -y -n napari-env python=3.8
+conda activate napari-env
+pip install napari[all]
 ```
 
 ### install from the master branch on Github
-To get the most up to date version of through pip call
+
+To install the "next-release" version from github via pip, call
```

Comment:
"next-release" is kinda ambigous, can this be phrased more clearly?

Reply:
Sure!  Suggestion? :)

Reply:
"bleeding-edge"? "nightly"?

Reply:
How about
"To install the current development version from github via pip"

Reply:
I can't edit this PR, otherwise I'd change now myself. I'm going to merge soon, as I don't like having instructions that are now out of date, but maybe we update this and consider updating the env instructions in a followup. Hope that's ok